### DISCUSSION TITLE
Welcome to Lux-Design-2022 Discussions!

### DISCUSSION THREAD

**Discussion by StoneT2000** [Lux-AI-Challenge/Lux-Design-S2#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SelwynVillers** [comment]
Are we ment to be able to view unit actions of both players? In the sample_obs.json file "action_queue":  is empty. Just wondering if you need to sample at a specific time point or not ment to have this information.

**Reply by StoneT2000** [reply]
Yes you should be able to see their action queues. However if the other unit only had one action that didn't repeat it won't show up there
